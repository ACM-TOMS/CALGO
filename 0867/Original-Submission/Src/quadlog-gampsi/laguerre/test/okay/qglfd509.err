
  ERROR: qglqfd() returns ierr =          4

  ERROR: qglqfd() returns ierr =          3
 Note: IEEE floating-point exception flags raised: 
    Inexact;  Invalid Operation; 
 See the Numerical Computation Guide, ieee_flags(3M) 
