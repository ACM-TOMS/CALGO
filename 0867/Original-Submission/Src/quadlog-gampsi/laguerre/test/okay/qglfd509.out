  Test integral: int(x^p * x^alpha * exp(-x), x = 0..infinity)

  1 ulp =   1.93E-34

  Floating-point precision appears to be (1 - log2(ulp) =) 113 bits

  alpha =  -2.0000000000000000  nquad =      1

  alpha =   1.0000000000000000  nquad =  32767

  Done
